lib/link_expansion/edition_diff.rb

```diff
@@ -8,21 +8,37 @@ def initialize(current_edition, version: nil, previous_edition: nil)
   end
 
   def field_diff
```

Comment:
Do we actually need a diff here, or do we just need to know that the old and new are unequal?

Reply:
We do, because we then compare the diff (which is just a list of fields which have changed) with "potential link expansion" fields - which is basically the top level list of fields which might change.

Only if there are fields that are present in both of those does it proceed with dependency resolution.

Reply:
Actually having thought about this, I think you're right. We can remove the `field_diff` method.

Reply:
@barrucadu Thanks for your review! I've added a commit to now just check if the hashes are different.